test/integration/DataDrivenSparqlTest.py

```diff
@@ -16,8 +16,8 @@ class DataDrivenSparqlTest(IntegrationTest):
     def setUp(self) -> None:
         super().setUp()
 
-        airport_queries = get_queries('sparql', 'epl')
-        for q in airport_queries:
+        epl_queries = get_queries('sparql', 'epl')
```

Comment:
Why are we moving to epl from airports?

Reply:
It was actually always using the EPL dataset, just with the wrong variable names. Changed it for clarity.